src/App.tsx

```diff
@@ -1,15 +1,17 @@
 import React from 'react'
-import { Route, Routes } from 'react-router-dom'
-
 import Map from '@pages/Map'
+import Navigation from '@pages/Navigation/Navigation'
+import { Route, Routes } from 'react-router-dom'
```

Comment:
(p5) 라이브러리 / 내부 코드 별로 import 순서를 정리하면 가독성이 좋아질 것 같습니다~

```suggestion
import React from 'react'
import { Route, Routes } from 'react-router-dom'

import Map from '@pages/Map'
import Navigation from '@pages/Navigation/Navigation'
```